Makefile

```diff
@@ -1,35 +1,59 @@
+# useful paths
+SRC_DIR = src
+HDR = $(SRC_DIR)/cs50.h
+SRC = $(SRC_DIR)/cs50.c
+
+BUILD_DIR = build
+OBJ = $(BUILD_DIR)/cs50.o
+USR_DIR = $(BUILD_DIR)/usr
+INCLUDE_DIR = $(USR_DIR)/include
+LIB_DIR = $(USR_DIR)/lib
+MAN_DIR = $(USR_DIR)/share/man/man3
+BUILD_SRC = $(USR_DIR)/src
+
+# eprintf
+EPRINTF_EXE = $(BUILD_DIR)/eprintf
```

Comment:
I didn't think it needed too. Just added for consistency, since I factored out other stuff. Felt it wouldn't look good to still have `build/*` hard-coded even in the tests. Can change, if you want!